operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,316 @@
+* [Procedure to delete a critical service from the ConfigMap](#procedure-to-delete-a-critical-service-from-the-configmap)
+* [Procedure to modify a critical service from the ConfigMap](#procedure-to-modify-a-critical-service-from-the-configmap)
+
+## Overview
+
+This page contains the procedures to list, add, delete and modify the critical services.
+
+The ConfigMap `rrs-mon-static` in the `rack-resiliency` namespace is where RR keeps its list of critical services. The API/CLI commands to add services end up adding 
+the new services to this ConfigMap. But because the API/CLI does not support edits or deletes, those can only be accomplished by directly editing the ConfigMap. 
+
+Please note that having two services with the same name in different namespaces is generally not considered a best practice in CSM, and as such, this use case is not supported in RRS. Additionally, services with the same name but different types (e.g., `statefulset` and `deployment`) are also not supported. Please ensure that this guideline is followed when adding a new critical service.
+
+## Procedure to add a critical service from the ConfigMap
+
+Refer "Updating critical services" section in [Adding critical service(s)](https://github.com/Cray-HPE/docs-csm/blob/CASM-5612/operations/rack_resiliency/rack_resiliency_cli.md#updating-critical-services)
+
+## Procedure to delete a critical service from the ConfigMap
+
+### 1. Verify that the ConfigMap exists
+
+```bash
+kubectl get ConfigMaps -n rack-resiliency
+```
+
+**Example output:**
+
+```text
+NAME                 DATA   AGE
+istio-ca-root-cert   1      6d21h
+kube-root-ca.crt     1      6d21h
+rrs-mon-dynamic      2      3d4h
+rrs-mon-static       11     3d4h
+```
+
+### 2. Verify that the list of critical services is present in the ConfigMap
+
+```bash
+kubectl describe configmap rrs-mon-static -n rack-resiliency
+```
+
+Example output:
+
+```text
+Name:         rrs-mon-static
+Namespace:    rack-resiliency
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+ceph_pre_monitoring_delay:
+----
+60
+critical-service-config.json:
+----
+{
+  "critical_services": {
+    "cilium-operator": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "coredns": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "cray-power-control-bitnami-etcd": {
+      "namespace": "services",
+      "type": "StatefulSet"
+    },
+    "cray-powerdns-manager": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-sls": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-spire-postgres": {
+      "namespace": "spire",
+      "type": "StatefulSet"
+    },
+    ...
+}
+...
+```
+
+### 3. Delete the critical service from the ConfigMap
+
+#### 3.1 Look for the critical service(s) to be deleted 
+
+Look inside the `critical-service-config.json` field, under the `data` section.
```

Comment:
This step and this instruction say to look for the services to be deleted, but then the command is to edit the configmap. I think the step should be to edit the configmap, and then there are substeps of what they should do once they are editing it -- specifically, locate the services to be deleted, and remove their stanzas.